framework/include/invokee.hpp

```diff
@@ -66,6 +66,9 @@ namespace gvk
 		/** Returns the name of this invokee */
 		const std::string& name() const { return mName; }
 
+		/** Sets the name of this invokee */
+		void set_name(const std::string aName) { mName = aName; }
```

Comment:
```suggestion
		void set_name(std::string aName) { mName = std::move(aName); }
```

Using `const std:string` (not a const reference) would force a copy in any case.